wbooks/src/app/components/BookList/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { FlatList, ListRenderItem } from 'react-native';
+import { IBook } from '@interfaces/BookInfo';
+import BookCard from '@components/BookCard';
+
+import styles from './styles';
+
+interface IProps {
+  bookArray: IBook[];
+}
+
+function BookList(props: IProps) {
+  const renderItem: ListRenderItem<IBook> = ({ item }: { item: IBook }) => <BookCard book={item} />;
+  return (
+    <FlatList<IBook>
+      style={styles.list}
+      contentContainerStyle={styles.listContainer}
+      data={props.bookArray}
+      renderItem={renderItem}
+      keyExtractor={item => item.id.toString()}
```

Comment:
Extract this function too (like you did with `renderItem`):
`const keyExtractor = ({ id }: number) => String(id);`

Reply:
Updated